File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex14.java
//Importando Scanner
import java.util.Scanner;

public class Ex14 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner input = new Scanner(System.in);
		
		System.out.println("N Pares e Ímpares\n\n");
		
		//vars
		int num, i, cont;
		
		//Entrada
		System.out.print("Digite a quantidade de termos: ");
		num = input.nextInt();
		
		
		//Saída
		System.out.println("\n" + num + " primeieros pares: ");
		cont=1;
		for( i=0; cont<=num; i++ )
		{
			if ( i%2 == 0 )
			{
				cont++;
				System.out.println(i);
			}
				
		}
		
		System.out.println("\n" + num + " primeieros pares: ");
		cont=1;
		for( i=1; cont<=num; i++ )
		{
			if ( i%2 == 1 )
			{
				cont++;
				System.out.println(i);
			}
				
		}
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex14.c
#include <stdio.h>

main()
{
    printf("N Primeiros Pares e Impares\n\n");

    //vars
    int n,cont=0, i=0;

    //Entrada
    printf("Insira a quantidade de pares e impares: ");
    scanf("%d", &n);

    printf("%d primeiros pares: \n",n);
    while(cont != n)
    {
        if(i%2 == 0)
        {
            printf("%d\n", i);
            cont++;
        }
        i++;
    }


    i = 1;
    cont=0;
    printf("\n%d primeiros impares: \n",n);
    while(cont != n)
    {
        if(i%2 == 1)
        {
            printf("%d\n", i);
            cont++;
        }
        i++;
    }

}
